sharktank/sharktank/types/tensors.py

```diff
@@ -250,6 +250,13 @@ def T(self) -> "InferenceTensor":
 
         return permute(self, dims=dims)
 
+    def __add__(
+        self, rhs: "sharktank.ops._registry.AnyTensor"
```

Comment:
Just leave off the type annotation. We're at the limit of what a useful annotation can do there.